products/monitoring/terraform.yaml

```diff
@@ -194,6 +194,9 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         is_set: true
       serviceLevelIndicator.basicSli.version: !ruby/object:Overrides::Terraform::PropertyOverride
         is_set: true
+      serviceLevelIndicator.basicSli.availability: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_expand: 'templates/terraform/custom_expand/monitoring_slo_availability_sli.go.erb'
```

Comment:
Can we remove this field in an encoder instead? This works, it's just more clear if we do it there. (This is a pretty nitpicky case- expanders are for converting a field to the API-expected value and encoders for cleaning up the request, and I feel this falls a little closer to the encoder case)

Reply:
In other resources i see only top level attributes are deleted in encoder. I am not sure how i can delete the sub-fields from the obj map. I want to send the `availability{}` block in the request just need to the delete `enabled`. Is it feasible in encoder ?